core/equations.gms

```diff
@@ -623,6 +623,26 @@ q_emiMac(t,regi,emiMac) ..
   )
 ;
 
+***--------------------------------------------------
+*' All CDR emissions summed up
+***--------------------------------------------------
+q_emiCdrAll(t,regi)..
+  vm_emiCdrAll(t,regi)
+       =e= !! BECC + DACC
+  (sum(emiBECCS2te(enty,enty2,te,enty3),vm_emiTeDetail(t,regi,enty,enty2,te,enty3))
+  + sum(teCCS2rlf(te,rlf), vm_ccs_cdr(t,regi,"cco2","ico2","ccsinje",rlf)))
+  !! scaled by the fraction that gets stored geologically
```

Comment:
I defined a parameter for that share, see [here](https://github.com/remindmodel/remind/blob/92dbf12c183cb2e8f337c9568a3ce1b31c6b81a9/core/postsolve.gms#L674) which would make it shorter.